flujo/domain/evaluation.py

```
@@ -11,3 +20,252 @@ class EvaluationScore(BaseModel):
+        objective_results: list[ValidationResult] = []
+        for validator in validators:
+            try:
+                validate_fn = getattr(validator, "validate", None) or validator
+                res = validate_fn(solution, context=context)
+                if inspect.isawaitable(res):
+                    res = await res
+                if isinstance(res, ValidationResult):
+                    objective_results.append(res)
+                elif hasattr(res, "is_valid"):
+                    payload: dict[str, object] = {
+                        "is_valid": bool(getattr(res, "is_valid")),
```

Ruff (0.14.10)

Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)